runtime/src/accounts_db.rs

```diff
@@ -753,8 +818,8 @@ impl AccountsDB {
             reclaims.extend(new_reclaims);
         }
 
-        drop(accounts_index);
         drop(storage);
+        drop(accounts_index);
```

Comment:
Maybe like this #8997... Sorry...

Reply:
yea, I think it makes it consistent, right?
always the order should be:
```
acquire(accounts_index)
acquire(stores)
drop(stores)
drop(accounts_index)
```
which is also the order if there is no explicit `drop`.

Reply:
Yeah, this got more consistent. I pondered on this a bit. I think there should be no problem in the order of unlocking ([ref](https://stackoverflow.com/a/53662845/10242004)); but, I think it's prudent to follow the RAII semantics/order as much as possible.